propka/run.py

```diff
@@ -28,6 +29,10 @@ def main(optargs=None):
        Removed ability to write out PROPKA input files.
     """
     # loading options, flags and arguments
+    logger = logging.getLogger("")
```

Comment:
Which messages are displayed by default – none, info or above, even debug... ? 

I think the default should be to show what the previous version of propka showed by default.

Reply:
It's controlled in the command-line settings:  `logging.INFO` is the default level like before.  

The statement above just grabs the root logger for setting handlers, etc.